neurom/features/__init__.py

```diff
@@ -64,15 +66,25 @@ def _flatten_feature(feature_shape, feature_value):
     return reduce(operator.concat, feature_value, [])
 
 
-def _get_neurites_feature_value(feature_, obj, neurite_filter, kwargs):
```

Comment:
I'm really weirded out by `kwargs` not being `**kwargs`, if it were, then `use_subtrees` could go in the right location

Reply:
Changed it into using `**kwargs`.